aurora.ipynb

```diff
@@ -78,7 +81,65 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "w_main = MainPanel()"
+    "title = f\"\"\"\n",
+    "    <div style=\"text-align: center;\">\n",
+    "        <h1>AURORA</h1>\n",
+    "        <p>Version {__version__}</p>\n",
+    "    </div>\n",
+    "\"\"\"\n",
+    "\n",
+    "w_header = ipw.VBox(\n",
+    "    layout={\n",
+    "        'width': '100%',\n",
+    "        'border': 'solid black 4px',\n",
+    "        'padding': '10px'\n",
+    "    },\n",
+    "    children=[ipw.HTML(title)],\n",
+    ")"
+   ]
```

Comment:
Is there any advantage of putting the code directly in the notebook inside all of these `"cell_type": "code"` boxes, instead to having all of these inside of a python file like it used to be? I know that, beforehand, the `MainPanel` was just one of the 3 in different notebooks. But now that you moved that to `ExperimentBuilder`, could we now turn `MainPanel` as the place where all of the code that is now in these cells gets put and just keep the notebook as minimal as possible?

Personally I think python files are easier to version control compared to notebooks, it makes it easier to error-check with the IDE, and all the `"` wrappings and necessary escapes (`\"\"\"`) make it a bit more noisy. It also feels a bit cleaner to only have the notebooks as a thin wrapper, only dealing with notebook setup, and not needing to mix in here parts of the app. But maybe I'm missing another critical reason to do it this way? 😅

Reply:
I agree. Especially since I already plan to move the CSS stuff to a static file. Sure, I'll move it to a `MainPanel` class. Good idea!

Reply:
Modified and rebased. Give it a looksee.